pom.xml

```diff
@@ -44,6 +44,8 @@
         <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
         <java.version>11</java.version>
         <kiwigrid.version>2.5</kiwigrid.version>
+        <log4j.version>2.17.1</log4j.version>
+        <logstash-logback-encoder.version>5.1</logstash-logback-encoder.version>
```

Comment:
Check maven repo and update with latest version, otherwise Vulnerabilities might get added.